static/js/snap-pages/components/PublishedSnapList/PublishedSnapList.tsx

```diff
@@ -0,0 +1,138 @@
+import {
+  Link,
+  MainTable,
+  Pagination,
+  Strip,
+} from "@canonical/react-components";
+import { ISnap } from "../../types";
+import { SnapNameEntry } from "./SnapNameEntry";
+import { NewSnapNotification } from "../NewSnapNotification";
+import { EmptySnapList } from "../EmptySnapList";
+import { PAGE_NUMBER } from "../../types/constants";
+
+export const PublishedSnapList = ({
```

Comment:
I think this should be `function PublishedSnapList` and the export at the bottom of the file to match other React components